scripts/common.py

```diff
@@ -109,16 +109,24 @@ class Config:
     Reflect changes to all configuration variables that depend on LLVM_VER
     """
     def _init_llvm_ver_deps(self):
+        def llvm_major_ver_ge(test_ver) -> bool:
+            (major, _, _) = self.LLVM_VER.split(".")
+            return int(major) >= test_ver
```

Comment:
I think `llvm_major_ver() -> int` would be better, and do the `>=` at call site.  It's much easier to read and more flexible, and `test_ver` isn't otherwise used in `llvm_major_ver_ge`.